client/src/components/ResultsProjector/ResultsProjector.jsx

```diff
@@ -130,13 +134,18 @@ function ResultsProjector({
         setStatus(STATUS.SHOWING);
         setTopResultIndex((topResultIndex) => {
           const newIndex = topResultIndex + getNumberOfRows();
-          return newIndex >= nonemptyResults.length ? 0 : newIndex;
+          return newIndex > nonemptyResults.length ||
+            (forecastView &&
```

Comment:
Let's document the behaviour with a comment:

```suggestion
            // When forecast view is enabled, the focus is usually on the advancing
            // results, so we only show a single page of non-advancing results and
            // roll back to the first page afterwards.
            (forecastView &&
```